src/types/dataset.ts

```diff
@@ -20,6 +20,7 @@ export type DatasetType = {
   identifier: string,
   describedBy?: any, // TODO
   describedByType?: any, //TODO
+  modified: string,
```

Comment:
This has added some typescript errors to /src/services/useMetastoreDataset.tsx

Reply:
feedback addressed:

https://github.com/GetDKAN/cmsds-open-data-components/pull/182/commits/b70571dfa474f49746574035eaf652ac6d696064